docs/content/config/advanced/user-patches.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Custom user changes & patches'
+---
+
+If you'd like to change, patch or alter files or behavior of `docker-mailserver`, you can use a script. Just place it the `config/` folder that is created on startup and call it `user-patches.sh`. The setup is done like this:
```

Comment:
`/config` not created on startup. It's already there when you clone this repo.

Reply:
It's only there if you clone the repo. I think there are users who clone the repo and there are users that follow the instructions in the usage section, `wget`-ing the scripts etc.

Nevertheless, how should we go about this?

Reply:
I edited the page. It should now be very clear.

Reply:
You are right. I forgot about the other method, because I always have used `git clone`.